src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private double sideB;
+
+    public Rectangle(Colors color, double side, double sideB, double height) {
+        super(color, side);
+        this.sideB = sideB;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: Isosceles Trapezoid, area: "
```

Comment:
The message in the `draw()` method is incorrect. It should be 'Rectangle', not 'Isosceles Trapezoid'.